src/utils/array.js

```diff
@@ -41,23 +41,27 @@ export function areEqualOrdered(arr1, arr2) {
  *
  * @param {!Array<T>} array
  * @param {function(T, number, !Array<T>):boolean} shouldRemove
+ * @return {!Array<T>}
  * @template T
  */
 export function remove(array, shouldRemove) {
+  const removed = [];
```

Comment:
Funnily enough, I actually removed this because no one was using the return value...